hail/src/main/scala/is/hail/expr/ir/TableWriter.scala

```diff
@@ -101,7 +148,111 @@ case class PartitionNativeWriter(spec: AbstractTypedCodecSpec, partPrefix: Strin
     eltType: PStruct,
     mb: EmitMethodBuilder[_],
     region: Value[Region],
-    stream: SizedStream): EmitCode = ???
+    stream: SizedStream): EmitCode = {
```

Comment:
Feel free to push back, this looks good, but is there any reason to not use the `scoped` methods for EmitCodeBuilder for a lot of this since we have the necessary EmitMethodBuilder to create the EmitCodeBuilder

Reply:
No, good call. I forgot that that existed. I'll refactor this.